src/components/Experiment/QuickInput/Tabs/SampleInput/SampleInputsTab.js

```diff
@@ -62,10 +64,12 @@ export default function SampleInputsTab(props) {
     }
 
     const task = Task.getStaticTask(props.task);
-    const sampleInputs = props.sampleInputs ?? [];
+    const sampleInputs = task.useMultiInput ? props.sampleInputs[props.inputIndex] : (props.sampleInputs ?? []);
```

Comment:
This and `useSampleInputControl` are the two examples of how we can refactor to support multiple inputs while not breaking the existing format for older tasks.